modules/installation-configuration-parameters.adoc

```diff
@@ -927,10 +927,14 @@ Additional Azure configuration parameters are described in the following table:
 |====
 |Parameter|Description|Values
 
-|`controlPlane.platform.azure.osDisk.diskSizeGB`
+|`controlPlane.platform.azure.osDisk.diskSizeGB`, `compute.platform.azure.osDisk.diskSizeGB`, `platform.azure.osDisk.diskSizeGB`
 |The Azure disk size for the VM.
 |Integer that represents the size of the disk in GB. The minimum supported disk size is `1024`.
 
+|`controlPlane.platform.azure.osDisk.diskType`, `compute.platform.azure.osDisk.diskType`, `platform.azure.osDisk.diskType`
+|`diskType` defines the type of disk.
```

Comment:
Not critical, but I'd recommend not repeating the name of the parameter. And instead just saying "The type of disk" or similar, to be consistent with the other entries